src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -982,7 +982,15 @@ Namespace Microsoft.CodeAnalysis.Semantics
         Private Function CreateBoundBlockOperation(boundBlock As BoundBlock) As IBlockStatement
             Dim statements As Lazy(Of ImmutableArray(Of IOperation)) = New Lazy(Of ImmutableArray(Of IOperation))(
                 Function()
-                    Return boundBlock.Statements.Select(Function(n) Create(n)).Where(Function(s) s.Kind <> OperationKind.None).ToImmutableArray()
+                    Return boundBlock.Statements.Where(
+                            Function(statement)
+                                ' Don't include the method/operator/accessor declaration in the block
+                                Return statement.Syntax.Kind() <> SyntaxKind.OperatorStatement AndAlso
+                                       statement.Syntax.Kind() <> SyntaxKind.FunctionStatement AndAlso
+                                       statement.Syntax.Kind() <> SyntaxKind.GetAccessorStatement AndAlso
+                                       statement.Syntax.Kind() <> SyntaxKind.SetAccessorStatement
```

Comment:
>statement.Syntax.Kind() <> SyntaxKind.SetAccessorStatement [](start = 39, length = 58)

What about AddHandlerAccessorStatement, RemoveHandlerAccessorStatement and RaiseEventAccessorStatement? It feels like we cannot really be sure what impact this change is going to have. What are we trying to accomplish and how is it related to the ```fixed``` statement?